code/game/machinery/computer/crew.dm

```diff
@@ -35,7 +35,8 @@ GLOBAL_DATUM_INIT(crewmonitor, /datum/crewmonitor, new)
 		// 00: Manager
 		"Manager" = 00,
 		"Extraction Officer" = 01,
-		"Records Officer" = 01,
+		"Records Officer" = 02,
```

Comment:
2 ROs?

Reply:
It's just where they are on the crew monitor console